CMakeLists.txt

```diff
@@ -153,7 +153,7 @@ endif()
 
 # Game VM modules are built with a recursive invocation of CMake, by which all the configuration
 # options are lost, except ones we explicitly choose to pass.
-set(DEFAULT_NACL_VM_INHERITED_OPTIONS
+set(DEFAULT_VMS_INHERITED_OPTIONS
```

Comment:
Can we drop the variable renaming changes (`VM_INHERITED_OPTIONS`, `VMS_PROJECTS`) that force Unvanquished CMakeLists.txt changes? Given than NaCl remains the only relevant type of VM for the near future, the current names seem clearer as to the variables' purpose. And the annoying mystery errors due to un-updated submodules are not really worth it for a minor refactoring change like this.

Reply:
I have dropped the renaming, this can be postponed for later.

Reply:
It means only this PR is now needed (not the game one).

Reply:
The variable NACL_VMS_PROJECTS which is used by the other CMakeLists also needs to preserve its original name

Reply:
Renamed back.